moto/guardduty/models.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import unicode_literals
+from boto3 import Session
+from moto.core import BaseBackend, BaseModel
+from datetime import datetime
+from uuid import uuid4
+from .exceptions import MissingParameterError
+
+
+class GuardDutyBackend(BaseBackend):
+    def __init__(self, region_name=None):
+        super(GuardDutyBackend, self).__init__()
+        self.region_name = region_name
+        self.detectors = {}
+
+    def reset(self):
+        region_name = self.region_name
+        self.__dict__ = {}
+        self.__init__(region_name)
+
+    def create_detector(
+        self, enable, client_token, finding_publishing_frequency, data_sources, tags
+    ):
+        if finding_publishing_frequency not in [
+            "FIFTEEN_MINUTES",
+            "ONE_HOUR",
+            "SIX_HOURS",
+        ]:
+            finding_publishing_frequency = "SIX_HOURS"
+
+        if enable is None or "":
+            raise MissingParameterError(parameter="enabled")
```

Comment:
Would be good to have a test for this scenario as well, using the pytest.raises paradigm ([Example](https://github.com/spulec/moto/blob/master/tests/test_cognitoidp/test_cognitoidp.py#L607))